packages/cli/tests/cli.ts

```diff
@@ -34,40 +34,70 @@ describe('cli', () => {
 				'mdsvex',
 				'paraglide=languageTags:en,es+demo:yes',
 				'mcp=ide:claude-code,cursor,gemini,opencode,vscode,other+setup:local'
+				// 'storybook' // No storybook addon during tests!
 			]
 		}
 	];
 
-	it.for(testCases)('should create a new project with name $projectName', async (testCase) => {
-		const { projectName, args } = testCase;
-		const svBinPath = path.resolve(monoRepoPath, 'packages', 'cli', 'dist', 'bin.js');
-		const testOutputPath = path.resolve(monoRepoPath, '.test-output', 'cli', projectName);
+	it.for(testCases)(
+		'should create a new project with name $projectName',
+		{ timeout: 10_000 },
+		async (testCase) => {
+			const { projectName, args } = testCase;
+			const svBinPath = path.resolve(monoRepoPath, 'packages', 'cli', 'dist', 'bin.js');
+			const testOutputPath = path.resolve(monoRepoPath, '.test-output', 'cli', projectName);
 
-		const result = await exec(
-			'node',
-			[
-				svBinPath,
-				'create',
-				testOutputPath,
-				'--template',
-				'minimal',
-				'--types',
-				'ts',
-				'--no-install',
-				...args
-			],
-			{ nodeOptions: { stdio: 'ignore' } }
-		);
+			const result = await exec(
+				'node',
+				[
+					svBinPath,
+					'create',
+					testOutputPath,
+					'--template',
+					'minimal',
+					'--types',
+					'ts',
+					'--no-install',
+					...args
+				],
+				{ nodeOptions: { stdio: 'pipe' } }
+			);
 
-		// cli finished well
-		expect(result.exitCode).toBe(0);
+			// cli finished well
+			expect(result.exitCode).toBe(0);
 
-		// test output path exists
-		expect(fs.existsSync(testOutputPath)).toBe(true);
+			// test output path exists
+			expect(fs.existsSync(testOutputPath)).toBe(true);
 
-		// package.json has a name
-		const packageJsonPath = path.resolve(testOutputPath, 'package.json');
-		const packageJson = parseJson(fs.readFileSync(packageJsonPath, 'utf-8'));
-		expect(packageJson.name).toBe(projectName);
-	});
+			// package.json has a name
+			const packageJsonPath = path.resolve(testOutputPath, 'package.json');
+			const packageJson = parseJson(fs.readFileSync(packageJsonPath, 'utf-8'));
+			expect(packageJson.name).toBe(projectName);
+
+			const snapPath = path.resolve(
+				monoRepoPath,
+				'packages',
+				'cli',
+				'tests',
+				'snapshots',
+				projectName
+			);
+			const relativeFiles = fs.readdirSync(testOutputPath, { recursive: true }) as string[];
+			for (const relativeFile of relativeFiles) {
+				if (!fs.statSync(path.resolve(testOutputPath, relativeFile)).isFile()) continue;
+				if (['.svg', '.env'].some((ext) => relativeFile.endsWith(ext))) continue;
```

Comment:
any reason for not comparing them? i mean it's likely they won't change much and will probably not provide any notable benefit, but I wonder what's more confusing - just adding them - or ignoring them

Reply:
For `.env`, since they are not committed, but generated, then they are part of test but not snapshot, so it's failing the test. I started to tweak gitignore, to have it committed there, but it's against the generated gitignore... So we would have to tweak this as well... All in all, I think skipping just the `.env` is a smarter solution.

For the `.svg`, first I think that we don't really care about the content of the svg in `sv`, second, I started to play with storybook, and there, I had some `png`, `svg` and co... And it was taking a lot of time. So it was mainly to speedup things.